src/cmake/sources.cmake

```diff
@@ -0,0 +1,313 @@
+    controller.cpp
+    cryptosettings.cpp
+    curve25519.cpp
+    dnshelper.cpp
+    dnspingsender.cpp
+    errorhandler.cpp
+    featurelist.cpp
+    filterproxymodel.cpp
+    fontloader.cpp
+    hacl-star/Hacl_Chacha20.c
+    hacl-star/Hacl_Chacha20Poly1305_32.c
+    hacl-star/Hacl_Curve25519_51.c
+    hacl-star/Hacl_Poly1305_32.c
+    hawkauth.cpp
+    hkdf.cpp
+    iaphandler.cpp
+    imageproviderfactory.cpp
+    inspector/inspectorhandler.cpp
+    inspector/inspectoritempicker.cpp
+    inspector/inspectorutils.cpp
+    inspector/inspectorwebsocketconnection.cpp
+    inspector/inspectorwebsocketserver.cpp
+    ipaddress.cpp
+    ipaddresslookup.cpp
+    itempicker.cpp
+    leakdetector.cpp
+    localizer.cpp
+    logger.cpp
+    loghandler.cpp
+    logoutobserver.cpp
+    main.cpp
+    models/device.cpp
+    models/devicemodel.cpp
+    models/feature.cpp
+    models/feedbackcategorymodel.cpp
+    models/guide.cpp
+    models/guideblock.cpp
+    models/guidemodel.cpp
+    models/helpmodel.cpp
+    models/keys.cpp
+    models/licensemodel.cpp
+    models/server.cpp
+    models/servercity.cpp
+    models/servercountry.cpp
+    models/servercountrymodel.cpp
+    models/serverdata.cpp
+    models/supportcategorymodel.cpp
+    models/survey.cpp
+    models/surveymodel.cpp
+    models/tutorial.cpp
+    models/tutorialmodel.cpp
+    models/user.cpp
+    models/whatsnewmodel.cpp
+    mozillavpn.cpp
+    networkmanager.cpp
+    networkrequest.cpp
+    networkwatcher.cpp
+    notificationhandler.cpp
+    pinghelper.cpp
+    pingsender.cpp
+    pingsenderfactory.cpp
+    platforms/dummy/dummyapplistprovider.cpp
+    platforms/dummy/dummyiaphandler.cpp
+    platforms/dummy/dummynetworkwatcher.cpp
+    platforms/dummy/dummypingsender.cpp
+    qmlengineholder.cpp
+    releasemonitor.cpp
+    rfc/rfc1112.cpp
+    rfc/rfc1918.cpp
+    rfc/rfc4193.cpp
+    rfc/rfc4291.cpp
+    rfc/rfc5735.cpp
+    serveri18n.cpp
+    settingsholder.cpp
+    simplenetworkmanager.cpp
+    statusicon.cpp
+    tasks/account/taskaccount.cpp
+    tasks/adddevice/taskadddevice.cpp
+    tasks/authenticate/taskauthenticate.cpp
+    tasks/captiveportallookup/taskcaptiveportallookup.cpp
+    tasks/deleteaccount/taskdeleteaccount.cpp
+    tasks/getfeaturelist/taskgetfeaturelist.cpp
+    tasks/controlleraction/taskcontrolleraction.cpp
+    tasks/createsupportticket/taskcreatesupportticket.cpp
+    tasks/function/taskfunction.cpp
+    tasks/group/taskgroup.cpp
+    tasks/heartbeat/taskheartbeat.cpp
+    tasks/ipfinder/taskipfinder.cpp
+    tasks/products/taskproducts.cpp
+    tasks/release/taskrelease.cpp
+    tasks/removedevice/taskremovedevice.cpp
+    tasks/sendfeedback/tasksendfeedback.cpp
+    tasks/servers/taskservers.cpp
+    tasks/surveydata/tasksurveydata.cpp
+    taskscheduler.cpp
+    telemetry.cpp
+    theme.cpp
+    timersingleshot.cpp
+    update/updater.cpp
+    update/versionapi.cpp
+    urlopener.cpp
+)
+
+# VPN Client header files
+target_sources(mozillavpn PRIVATE
```

Comment:
we can finally merge SOURCES+HEADERS. Let's do it!

Reply:
Yes, we can indeed merge them. I just opted not to in order to try and preserve similarities with the `*.qmake` files for now (which helps a lot when rebasing this PR with upstream changes)

So, I'm going to opt to merge them later only when we get around to dropping support for `qmake`